lib/routes/user/marketplace/marketplace.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import 'package:breez/bloc/blocs_provider.dart';
+import 'package:breez/bloc/marketplace/marketplace_bloc.dart';
+import 'package:breez/bloc/marketplace/vendor_model.dart';
+import 'package:breez/routes/user/marketplace/vendor_row.dart';
+import 'package:breez/widgets/back_button.dart' as backBtn;
+import 'package:breez/theme_data.dart' as theme;
+
+class MarketplacePage extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() {
+    return new MarketplacePageState();
+  }
+}
+
+class MarketplacePageState extends State<MarketplacePage> {
+  final String _title = "Marketplace";
+  final GlobalKey<ScaffoldState> _scaffoldKey = new GlobalKey<ScaffoldState>();
+  MarketplaceBloc _marketplaceBloc;
+  bool _isInit = false;
+
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  void didChangeDependencies() {
+    if (!_isInit) {
+      _marketplaceBloc = AppBlocsProvider.of<MarketplaceBloc>(context);
+      _isInit = true;
+    }
+    super.didChangeDependencies();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return StreamBuilder(
+        stream: _marketplaceBloc.vendorsStream,
+        builder: (context, snapshot) {
+          List<VendorModel> vendorsModel;
+          if (snapshot.hasData) {
```

Comment:
no need for the "if" statement. Just an assignment like this should work:
`List<VendorModel> vendorsModel = snapshot.data`